dhall-json/json-to-dhall/Main.hs

```diff
@@ -11,22 +11,17 @@ import Control.Exception   (SomeException, throwIO)
 import Data.Text           (Text)
 import Data.Version        (showVersion)
 import Dhall.JSONToDhall
+import Dhall.Util          (renderExpression)
```

Comment:
I think we've been using import lists mostly for types and common functions, mostly from `base`.

For an "uncommon" utility like this one I'd suggest rather using a qualified import.

Reply:
Thanks! I didn't know, I'll fix it right away